### PR TITLE
chore: changelog for `5.4.1`

### PR DESCRIPTION
**Pull request by susiwen8**

### CODE REVIEW

File: en/changelog.md [hunk 1]
```diff
@@ -1,3 +1,31 @@
+## 5.4.1
```
Comment (plainheart) [inline en/changelog.md:1]: Missing `v` before `5.4.1`

File: en/changelog.md [hunk 2]
```diff
@@ -1,3 +1,31 @@
+## 5.4.1
+
++ feat(i18n): add Hungarian(hu_HU) translation [#17978](https://github.com/apache/echarts/issues/17978) ([ifjkovacsik](https://github.com/ifjkovacsik))
```
Comment (plainheart) [inline en/changelog.md:3]: The format should be like `[type] [scope]: message`.

**Review by plainheart — CHANGES_REQUESTED** [review]

File: en/changelog.md [hunk 3]
```diff
@@ -1,3 +1,30 @@
...
++ [Fix] [type]: add `string` & `number` type into the `center` option of pie series [#17917](https://github.com/apache/echarts/issues/17917) ([plainheart](https://github.com/plainheart))
++ [Fix] [heatmap]: fix heatmap label may overlap when `dataZoom` is enabled [#17876](https://github.com/apache/echarts/issues/17876) ([plainheart](https://github.com/plainheart))
++ [Fix] [tooltip]: fix tooltip arrow may cover the content when border is wide [#17875](https://github.com/apache/echarts/issues/17875) ([plainheart](https://github.com/plainheart))
++ [Fix] [i18n]: language code for Ukrainian is UK, not UA (follow-up) [#17812](https://github.com/apache/echarts/issues/17812) ([striezel](https://github.com/striezel))
++ [Fix] [type]: add overflow property to AxisLabelBaseOption interface. close #17363 [#17808](https://github.com/apache/echarts/issues/17808) ([breizh24](https://github.com/breizh24))
++ [Fix] [lines)]: add missing `function` callback into `lineStyle.color` for lines series. [#17780](https://github.com/apache/echarts/issues/17780) ([plainheart](https://github.com/plainheart))
```
Comment (plainheart) [inline en/changelog.md:16]: The scope of this change should be `type` rather than `lines` and please put this item below.

File: en/changelog.md [hunk 4]
```diff
@@ -1,3 +1,30 @@
...
++ [Fix] [dataZoom]: fallback to extent start/end when value or percent is invalid [#17934](https://github.com/apache/echarts/issues/17934) ([plainheart](https://github.com/plainheart))
++ [Fix] [pie]: fix `center` may not be converted to correct point when coordinate system is specified [#17920](https://github.com/apache/echarts/issues/17920) ([plainheart](https://github.com/plainheart))
++ [Fix] [type]: add `string` & `number` type into the `center` option of pie series [#17917](https://github.com/apache/echarts/issues/17917) ([plainheart](https://github.com/plainheart))
++ [Fix] [heatmap]: fix heatmap label may overlap when `dataZoom` is enabled [#17876](https://github.com/apache/echarts/issues/17876) ([plainheart](https://github.com/plainheart))
++ [Fix] [tooltip]: fix tooltip arrow may cover the content when border is wide [#17875](https://github.com/apache/echarts/issues/17875) ([plainheart](https://github.com/plainheart))
++ [Fix] [i18n]: language code for Ukrainian is UK, not UA (follow-up) [#17812](https://github.com/apache/echarts/issues/17812) ([striezel](https://github.com/striezel))
```
Comment (plainheart) [inline en/changelog.md:14]: The message of this item needs to be updated. https://github.com/apache/echarts/pull/17812

File: en/changelog.md [hunk 5]
```diff
@@ -1,3 +1,30 @@
...
++ [Fix] [type]: add overflow property to AxisLabelBaseOption interface. close #17363 [#17808](https://github.com/apache/echarts/issues/17808) ([breizh24](https://github.com/breizh24))
```
Comment (plainheart) [inline en/changelog.md:15]: It's good to wrap the keyword `overflow` & `AxisLabelBaseOption` with \`

File: en/changelog.md [hunk 6]
```diff
@@ -1,3 +1,30 @@
...
++ [Fix] [pie]: fix percent isn't updated after the legend is changed [#17734](https://github.com/apache/echarts/issues/17734) ([plainheart](https://github.com/plainheart))
++ [Fix] [type]: EChartsInitOpts type error about height and width. close #17667 [#17727](https://github.com/apache/echarts/issues/17727) ([MainHou](https://github.com/MainHou))
++ [Fix] [pie]: fix `labelLine` NPE when `minShowLabelRadian` is set [#17715](https://github.com/apache/echarts/issues/17715) ([plainheart](https://github.com/plainheart))
++ [Fix] [gauge]: add gauge sectors in the order of the "clockwise" option [#17691](https://github.com/apache/echarts/issues/17691) ([ZeekoZhu](https://github.com/ZeekoZhu))
++ [Fix] [timeline]: fix chart can't work when axis line is not shown [#17647](https://github.com/apache/echarts/issues/17647) ([plainheart](https://github.com/plainheart))
++ [Fix] [universalTransition]: incorrect dataGroupId for old data items in universalTransition [#17559](https://github.com/apache/echarts/issues/17559) ([tyn1998](https://github.com/tyn1998))
```
Comment (plainheart) [inline en/changelog.md:22]: It's better to set the scope to `animation`

File: en/changelog.md [hunk 7]
```diff
@@ -1,3 +1,30 @@
+## v5.4.1
+
++ [Feature] [i18n] Add Hungarian(hu_HU) translation [#17978](https://github.com/apache/echarts/issues/17978) ([ifjkovacsik](https://github.com/ifjkovacsik))
++ [Feature] [svg] Support gradients and patterns for background color in SVG renderer [#959](https://github.com/ecomfe/zrender/issues/959) ([plainheart](https://github.com/plainheart)
```
Comment (plainheart) [inline en/changelog.md:4]: Missing `)` in the end

File: en/changelog.md [hunk 8]
```diff
@@ -1,3 +1,30 @@
...
++ [Fix] [animation] Fix incorrect `dataGroupId` for old data items in universalTransition [#17559](https://github.com/apache/echarts/issues/17559) ([tyn1998](https://github.com/tyn1998))
++ [Fix] [legend] Fix legend `inverse` doesn't work if series data have the same name [#17510](https://github.com/apache/echarts/issues/17510) ([caiwukun](https://github.com/caiwukun))
++ [Fix] [ssr] Fix wrong expression for extracting the font size [#968](https://github.com/ecomfe/zrender/issues/968) ([plainheart](https://github.com/plainheart))
++ [Fix] [marker] Fix `markArea` doesn't show when using `x/y` pixel but `coord` is not defined [#17217](https://github.com/apache/echarts/issues/17217) ([jiawulin001](https://github.com/jiawulin001))
++ [Fix] [markArea] `MarkArea` of bar series now covers whole specified categories [#17098](https://github.com/apache/echarts/issues/17098) ([jiawulin001](https://github.com/jiawulin001))
++ [Fix] [type] EChartsInitOpts type error about height and width. close #17667 [#17727](https://github.com/apache/echarts/issues/17727) ([MainHou](https://github.com/MainHou))
```
Comment (plainheart) [inline en/changelog.md:23]: Wrap the `EChartsInitOpts` and add **'Fix'** before it

File: en/changelog.md [hunk 9]
```diff
@@ -1,3 +1,30 @@
+## v5.4.1
+
++ [Feature] [i18n] Add Hungarian(hu_HU) translation. [#17978](https://github.com/apache/echarts/issues/17978) ([ifjkovacsik](https://github.com/ifjkovacsik))
++ [Feature] [svg] Support gradients and patterns for background color in SVG renderer. [#959](https://github.com/ecomfe/zrender/issues/959) ([plainheart](https://github.com/plainheart)
++ [Feature] [candlestick] Provide borderColorDoji option for custom doji color. [#17717](https://github.com/apache/echarts/issues/17717) ([Ovilia](https://github.com/Ovilia))
```
Comment (plainheart) [inline en/changelog.md:5]: Wrap `borderColorDoji`

File: en/changelog.md [hunk 10]
```diff
@@ -1,3 +1,30 @@
...
++ [Fix] [marker] Fix `markArea` doesn't show when using `x/y` pixel but `coord` is not defined [#17217](https://github.com/apache/echarts/issues/17217) ([jiawulin001](https://github.com/jiawulin001))
++ [Fix] [markArea] `MarkArea` of bar series now covers whole specified categories [#17098](https://github.com/apache/echarts/issues/17098) ([jiawulin001](https://github.com/jiawulin001))
++ [Fix] [type] EChartsInitOpts type error about height and width. close #17667 [#17727](https://github.com/apache/echarts/issues/17727) ([MainHou](https://github.com/MainHou))
++ [Fix] [type] Add `overflow` property to `AxisLabelBaseOption` interface. close #17363 [#17808](https://github.com/apache/echarts/issues/17808) ([breizh24](https://github.com/breizh24))
++ [Fix] [type] Add `string` & `number` type into the `center` option of pie series [#17917](https://github.com/apache/echarts/issues/17917) ([plainheart](https://github.com/plainheart))
++ [Fix] [type] Add missing `function` callback into `lineStyle.color` for lines series. [#17780](https://github.com/apache/echarts/issues/17780) ([plainheart](https://github.com/plainheart))
```
[the remark this answers is not in the document]
Comment (susiwen8) [reply en/changelog.md:28]: Sure

**Review by Ovilia — APPROVED** [review]
LGTM

**Review by plainheart — APPROVED** [review]

**Pull request merged**